Adafruit_HX8357.cpp

```diff
@@ -14,9 +14,20 @@
  ****************************************************/
 
 #include "Adafruit_HX8357.h"
-#include <limits.h>
 
-#include <SPI.h>
+#if defined(PLATFORM_ID)  //Only defined if a Particle device
```

Comment:
It looks like a SPARK define is still in the code - I suggest switching this to #if defined(SPARK) to make it clear that the code inside is only for the SPARK. "PLATFORM_ID" is not very descriptive.